core/src/main/java/org/elasticsearch/index/fielddata/ScriptDocValues.java

```diff
@@ -291,4 +291,38 @@ public double geohashDistanceWithDefault(String geohash, double defaultValue) {
             return geohashDistance(geohash);
         }
     }
+
+    class Booleans extends AbstractList<Boolean> implements ScriptDocValues<Boolean> {
```

Comment:
static is not necessary since it is inside an interface, but good catch on the final.